README.md

```diff
@@ -33,6 +33,15 @@ If you help to develop Lunarvim, you can install a specific branch branch direct
 LVBRANCH=rolling bash <(curl -s https://raw.githubusercontent.com/ChristianChiarulli/lunarvim/rolling/utils/installer/install.sh)
 ```
 
+If the your installation is stuck with `Ok to remove? [y/N]`, it means there are some leftovers, \
```

Comment:
Just a grammar suggestion:

```diff
- If the your installation is stuck with
+ If your installation is stuck on
```

May also want to start a new sentence after "leftovers".

Reply:
thank you for suggestion